saq/email.py

```diff
@@ -18,8 +19,13 @@
 def normalize_email_address(email_address):
     """Returns a normalized version of email address.  Returns None if the address cannot be parsed."""
     name, address = parseaddr(email_address)
-    if address is None:
-        return None
+    if address is '':
+        # attempt to fix known cases the stdlib has, like <<person@example.com>>
+        while email_address and '<<' and '>>'  in email_address:
```

Comment:
Is this right? Did you mean to say '<<' in email address and '>>' in email_address

Reply:
nope, that's not right at all. Must be going cross eyed. Fixing